packages/orbit-design-tokens/src/dictionary/definitions/foundation/palette/ink.json

```diff
@@ -5,12 +5,14 @@
         "dark": { "value": "#252A31", "internal": true, "type": "color" },
         "dark-active": { "value": "#0B0C0F", "internal": true, "type": "color" },
         "dark-hover": { "value": "#181B20", "internal": true, "type": "color" },
+        "dark-transparency": { "value": "#252A311A", "internal": true, "type": "color" },
```

Comment:
so you added transparency tokens only for tests? 🤔

Reply:
No. Tokens exist for that. A primary text with background will have the class `.bg-text-primary-background` to apply the background

Reply:
Okay, got It. What you should change there, IMO -> the palette tokens should not include those transparency values, as we won't use them somewhere else. Just add them as static values to component level file text-background 🤔

Reply:
> And we can't define it on the component to be 10% of the opacity of the original token, can we?

Currently do not think so, except if you will use the rgba, but it still will be static 
Maybe a good idea to extend our tokens generation logic to support automatic conversion to transparency token

> Besides that, by having it defined as internal on the foundation, they are only part of the foundation, they're not available on the theme. Isn't the foundation supposed to be precisely for those situations (tokens that should exist for us but not for external use)

if it has an internal flag, then it will be only for us 

the problem with that is also that foundation colors are linked to Figma and Figma does not have those transparency tokens. So if we decide now to keep them as tokens of the palette and not statically for Text (I agree, that it might in future force us to manually change it) 

So we can just move them to global or have it statically saved as I suggested above and later add a feature that will transform the token automatically :)

Reply:
I think it is possible to transform tokens and apply opacity. I'll investigate and try to find a solution

Reply:
why not to use rgba value? 🤔 I mean simply to add support for our tokens package, something like: 
```
{ type: "color-rgba", value: {foundation.palette.ink.dark}, opacity: 10 }

under the hood: convertHexToRgba(token, opacity)

output will be rgba token
```

Reply:
Yes, that's what I ended up doing. I noticed we actually had some buggy internal transformation (for instance, `socialButtonAppleBorderColorFocus` should have transparency, but if you check on Docs and other token outputs it was missing the alpha value. So I also fixed that. I will add the fix on this branch on a separate commit, for the sake of simplifying